.github/workflows/main.yml

```
@@ -490,7 +504,14 @@ jobs:
       # dependabot PRs, but we still want all other tests above to run
       # in those cases.
       - name: Check for duplicate Node dependencies
-        run: yarn dedupe --check
+        run: |
+          yarn dedupe --check
```

There's no need to change this.

```suggestion
        run: yarn dedupe --check
```